ariadne/cardano/src/Ariadne/Wallet/Backend/Tx.hs

```diff
@@ -193,7 +193,9 @@ sendTx
                 ourAddresses
                 (map TxOutAux outs)
                 (const newChangeAddress)
-        liftIO $ whenLeftM (awlNewPending awl ourAccountId txAux) throwM
+        -- TODO: call newPending even when inputs are selected from multiple accounts.
```

Comment:
> Create an issue to implement it properly (with proper description).

Please do it and leave issue ID here. I'll approve after that.

Reply:
Still no issue ID here :thinking: